### PR TITLE
Support sourcehut

### PR DESCRIPTION
**Pull request by satotake**
related #20

[sourcehut](https://sourcehut.org) is an engineering platform by @ddevault which has git repository hosting service.

Although it is not mature yet (public alpha stage), it has some interesting features such as good-old email-oriented flow.

If you liked, please support sourcehut

Thanks

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -299,17 +299,20 @@ function parse(src) {
 		''
 	);
 	if (!supported.has(site)) {
-		throw new DegitError(`degit supports GitHub, GitLab and BitBucket`, {
-			code: 'UNSUPPORTED_HOST',
-		});
+		throw new DegitError(
+			`degit supports GitHub, GitLab, sourehut and BitBucket`,
```
Comment (YogliB) [inline src/index.js]: Should be "sourcehat"

Comment (ddevault) [reply src/index.js]: Should be "Sourcehut" :)

Comment (YogliB) [reply src/index.js]: ‏‏לכידה

Comment (ddevault) [reply src/index.js]: Well, it's certainly not "sourcehat" :stuck_out_tongue:

Comment (YogliB) [reply src/index.js]: You're right! :sweat_smile:

Comment (satotake) [reply src/index.js]: 👍

**Pull request merged**